airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -80,30 +84,26 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
     final int version = Workflow.getVersion(VERSION_LABEL, Workflow.DEFAULT_VERSION, CURRENT_VERSION);
     final String taskQueue = Workflow.getInfo().getTaskQueue();
 
-    // Temporarily suppressed to address OC issue #1210
-    @SuppressWarnings("PMD.UnusedLocalVariable")
     final int autoDetectSchemaVersion =
         Workflow.getVersion(AUTO_DETECT_SCHEMA_TAG, Workflow.DEFAULT_VERSION, AUTO_DETECT_SCHEMA_VERSION);
 
-    // Temporarily disabled to address OC issue #1210
```

Comment:
Just to double check, are all the issue related to this ticket solved? and is there a way to test it in dev?

Reply:
Yes, the config repository is removed from all activities (config fetch activity & refresh schema activity). I'm not sure how to test this in the data plane. @jdpgrailsdev do you know?

Reply:
We can at least test it locally to verify that the application starts up in "data plane" mode.  This will at least test the problem we saw in production with the missing data source.